src/index.tsx

```diff
@@ -129,45 +127,58 @@ export function useSpeechRecognition(options: SpeechRecognitionOptions = default
   );
 
   const onRecognitionDisconnect = useCallback(() => {
-    dispatch(setStatus(SpeechRecognitionStatus.STOPPED));
     if (pauseAfterDisconnect) {
-      dispatch(setListening(false));
+      dispatch(setStatus(SpeechRecognitionStatus.STOPPED));
```

Comment:
might be better to `dispatch(paused())` here then do whatever state mutation in the reducer.  Slightly more declarative

Reply:
I'm going to circle back to this one once I've gotten things taken care of for both repos

Reply:
Nvm. Circling back now since there is nothing on the other one yet.